app/models/concerns/github_pull_request.rb

```diff
@@ -63,6 +63,19 @@ def set_time_build_status
     end
   end
 
+  def set_cla_build_status
+    status = "groupthink/cla"
+    if cla_required?
+      if cla_accepted?
+        set_build_status(:success, I18n.t("build_status.cla.accepted"), status)
+      else
+        set_build_status(:pending, I18n.t("build_status.cla.pending"), status)
+      end
+    else
+      set_build_status(:success, I18n.t("build_status.cla.none"), status)
```

Comment:
Do we need to set the build status at all if there's no CLA? Probably not...

Reply:
Good idea, I hadn't thought of just not adding it, doh!